test/test_transformer.py

```diff
@@ -238,12 +238,6 @@ def test_itransform_no_error():
     list(itransform(pj, Proj(4326), [(pjx, pjy)], radians=True, errcheck=True))
 
 
-def test_transform_exception():
-    transformer = Transformer.from_proj("+init=epsg:4326", "+init=epsg:27700")
-    with pytest.raises(ProjError):
-        transformer.transform(100000, 100000, errcheck=True)
```

Comment:
Maybe keep the test but update the expected result?

Reply:
The purpose of the test is to check the errors raised by proj are caught properly. Maybe another method of failure not already used in the tests could be used here. Ideas welcome.

Reply:
If this still raises an error, maybe this can be used in its place: https://github.com/pyproj4/pyproj/issues/269.

Reply:
But what does the code in this test return now with proj 6.1 ?

Reply:
Currently nether raise an error:
```
>>> from pyproj import Transformer
>>> transformer = Transformer.from_proj("+init=epsg:4326", "+init=epsg:27700")
>>> transformer.transform(100000, 100000, errcheck=True)
(inf, inf)
>>> transformer = Transformer.from_proj("+init=epsg:26918", "+init=epsg:4326")
>>> transformer.transform(0, 0, errcheck=True)
(-79.48874388438705, 0.0)
```
So, I guess this means that another method for raising an error will need to be used to keep the test.

Reply:
I think it would also be good to keep this test (update with the new result), as it is relevant to test that we get an answer for out of area coordinates.

Reply:
Okay, keeping the test and modifying its purpose to check for `inf` instead of an error works for me.

Reply:
I just made the update.